rpcserver.go

```diff
@@ -6817,6 +6863,25 @@ func (r *rpcServer) ForwardingHistory(ctx context.Context,
 			AmtInMsat:   uint64(amtInMsat),
 			AmtOutMsat:  uint64(amtOutMsat),
 		}
+
+		if !req.SkipPeerAliasLookup {
+			resp.ForwardingEvents[i].PeerAliasIn, err =
+				getRemoteAlias(event.IncomingChanID)
+			if err != nil {
+				return nil, fmt.Errorf(
```

Comment:
Calls to logging or `fmt.Errorf` should stay formatted in the compact way to clearly visually distinguish them from normal function/method calls (see formatting guide).

Reply:
Thanks, I fixed your suggestions.